corehq/apps/locations/models.py

```diff
@@ -116,6 +116,18 @@ class LocationQueriesMixin(object):
     def location_ids(self):
         return self.values_list('location_id', flat=True)
 
+    def couch_locations(self, wrapped=True):
+        """
+        Returns the couch locations corresponding to this queryset.
+        If you're calling this, take a long hard look at yourself and ask why.
+        Do you really need couch locations here?
+        """
```

Comment:
Added, but I was having a hell of a time to actually get it to show up.  The only thing that works for me is to start the dev server with `$ python -Wd manage.py runserver`.  Do you actually see these warnings?
I tried the suggestions [here](https://code.djangoproject.com/ticket/18985) and added [this](https://github.com/django/django/pull/385/files) to our `manage.py`, and modified that same line in our `settings.py`, but it still doesn't show up unless I pass the flags to python or set the environment variable `export PYTHONWARNINGS="d"`.